src/cli/telemetry/helpers.ts

```diff
@@ -20,7 +20,7 @@ export const isInteractive = (sys: d.CompilerSystem, config: d.Config, object?:
       ci:
         ['CI', 'BUILD_ID', 'BUILD_NUMBER', 'BITBUCKET_COMMIT', 'CODEBUILD_BUILD_ARN'].filter(
           (v) => !!sys.getEnvironmentVar(v)
-        ).length > 0 || !!config?.flags?.ci,
```

Comment:
I think we swung a little too far here. If `flags` is `undefined` the property accessor of `.ci` binds more tightly than the double bang operator, so we'll get an error at runtime

Reply:
Considered in 9baee30